samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -173,6 +173,8 @@ Emergency data storage
 
 To build this sample with support for emergency data storage, set ``OVERLAY_CONFIG`` to :file:`overlay-emds.conf`.
 This will save replay protection list (RPL) data and some of the :ref:`bt_mesh_lightness_srv_readme` data to the emergency data storage instead of to the :ref:`settings_api`.
+When using EMDS, certain conciderations need to be taken with regards to hardware choices in your application design.
+See :ref:`emds_readme` for more information.
```

Comment:
This is probably a nit-picking, but if I as a user didn't know anything about EMDS, then I would start reading about it from the beginning. But if I knew about EMDS, then it would make more sense if this sentence point me to the exact subsection with the calculations instead of forcing me to find it among other text.

Reply:
I agree with Pavel.
If considerations we are talking about are all within the Application integration section in EMDS, then it would be good to add label for that section and reference it directly.